pkg/lib/cockpit-po-plugin.js

```diff
@@ -111,8 +109,19 @@ function init(options) {
 function run(webpack_module, webpack_compilation) {
     const promises = [];
     config.subdirs.map(subdir =>
```

Comment:
I'm getting a headache trying to read this.  Can we turn this into nested `for` loops or something instead?